pytest_splunk_addon/standard_lib/data_models/Authentication.json

```diff
@@ -1,145 +1,238 @@
-              "fields_cluster": [],
-              "fields": [],
-              "child_dataset": [],
-              "search_constraints": "action=\"failure\""
-            },
-            {
-              "name": "Successful_Privileged_Authentication",
-              "tags": [["authentication", "privileged"]],
-              "fields_cluster": [],
-              "fields": [],
-              "child_dataset": [],
-              "search_constraints": "action=\"success\""
-            }
-          ]
+            "name": "Authentication",
+            "tags": [
+                [
+                    "authentication"
+                ]
+            ],
+            "fields_cluster": [],
+            "fields": [
+                {
+                    "name": "action",
+                    "type": "required",
+                    "expected_values": [
+                        "success",
+                        "failure",
+                        "pending",
+                        "error"
+                    ],
+                    "comment": "The action performed on the resource."
+                },
+                {
+                    "name": "app",
+                    "type": "required",
+                    "comment": "The application involved in the event (such as ssh, splunk, win:local)."
+                },
+                {
+                    "name": "authentication_method",
+                    "type": "optional",
+                    "comment": "The method used to authenticate the request"
+                },
+                {
+                    "name": "authentication_service",
+                    "type": "optional",
+                    "comment": "The service used to authenticate the request"
+                },
+                {
+                    "name": "dest",
+                    "type": "required",
+                    "comment": "The target involved in the authentication. You can alias this from more specific fields, such as dest_host, dest_ip, or dest_nt_host."
+                },
+                {
+                    "name": "src_user_type",
+                    "type": "optional",
+                    "comment": "The type of the user who initiated the privilege escalation."
+                },
+                {
+                    "name": "user_type",
+                    "type": "optional",
+                    "comment": "The type of the user involved in the event or who initiated the event.\nIAMUser, Admin, or System."
+                },
+                {
+                    "name": "src_user_role",
+                    "type": "optional",
+                    "comment": "The role of the user who initiated the privilege escalation."
+                },
+                {
+                    "name": "duration",
+                    "type": "optional",
+                    "validity": "if(isnum(duration) and duration>0 and duration<3600, duration, null())",
```

Comment:
why would duration be limited to 0-3599? seems like it should be any interger >0.

Reply:
I think some kind of a cap should be set. Maybe higher than this, but the test script should not leave open ended values. The goal is to catch incorrectly extracted values. So if the auth duration is more than 1 hour, that at least should prompt the TA Dev to investigate what is going on, if the extraction done correctly, is this ok for the vendor to provide such long duration. That is the aim of the PSA, I think.